integrationtests/utils.go

```diff
@@ -184,12 +184,12 @@ func diskCleanup(t *testing.T, vhdxPath, mountPath, testPluginPath string) {
 	if out, err = runPowershellCmd(t, cmd); err != nil {
 		t.Errorf("Error: %v. Command: %s. Out: %s", err, cmd, out)
 	}
-	cmd = fmt.Sprintf("rmdir %s", mountPath)
+	cmd = fmt.Sprintf("rm -r -fo %s", mountPath)
```

Comment:
is there a reason for adding options -r -fo?

Reply:
not really, I just tried this to remove the test folders generated during diskInit, I reverted this change and instead made the change during `diskInit`

Reply:
I had to make this change because during teardown the test would sometimes flake, sometimes it attempted to delete a folder from a different stale test, while this wouldn't happen in a new VM I thought it'd be better to isolate the test to a single directory just like how it's going to happen with the csi-proxy flag update (from 2 flags to the flag `kubelet-path`)